### PR TITLE
Adding installation directions to README

### PR DESCRIPTION
**Pull request by rbharath**
This PR adds installation directions to the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,31 @@
...
+- The test suite uses `Geth` utilities and libraries. [Install Geth from source](https://github.com/ethereum/go-ethereum/wiki/Installation-Instructions-for-Ubuntu) in the right place in your Go environment.
+
+### Run Tests
+
+The test suites are written in Go, so you can run tests from the top level directory with
+```
+go test -v ./tests/...
+```
+
+You can also run individual tests by specifying a subdirectory.
```
Comment (robrobbins) [inline README.md]: this is the spot to use the raw go test command from above with a path to a package, so specifiy an example dir, like `ethertoken` or something...

File: README.md [hunk 2]
```diff
@@ -1,5 +1,31 @@
...
+
+### Making changes to contracts
+
+If you've changed a contract in this repo, you will need to regenerate the `.abi` and `.bin` files tied to that file. These files are used to regenerate the go API bindings for the contract. You finally need to generate the external vyper files. For example, let's suppose that you've changed the `Voting.by` contract. Then you would run these commands
```
Comment (robrobbins) [inline README.md]: `.by`? `Voting.go` is prob what you wanted (did i do that?)

File: README.md [hunk 3]
```diff
@@ -1,5 +1,31 @@
...
 
-#### Golang testing and sandbox for the Computable Protocol.
+## Golang testing and sandbox for the Computable Protocol.
 [![Build Status](https://travis-ci.org/computablelabs/goest.svg?branch=master)](https://travis-ci.org/computablelabs/goest)
 
+### Installation
+
+- First start by installing vyper (see [installation instructions](https://github.com/ethereum/vyper/blob/master/docs/installing-vyper.rst)).
+- Make sure your Go environment is configured (see [directions](https://golang.org/doc/install))
+- The test suite uses `Geth` utilities and libraries. [Install Geth from source](https://github.com/ethereum/go-ethereum/wiki/Installation-Instructions-for-Ubuntu) in the right place in your Go environment.
```
[the remark this answers is not in the document]
Comment (rbharath) [reply README.md]: Yep, will add a pointer to `go get`

File: README.md [hunk 4]
```diff
@@ -1,5 +1,31 @@
...
+
+### Run Tests
+
+The test suites are written in Go, so you can run tests from the top level directory with
+```
+go test -v ./tests/...
```
[the remark this answers is not in the document]
Comment (rbharath) [reply README.md]: Changed

File: README.md (hunk 2 above)
Comment (rbharath) [reply README.md]: Oops, meant `Voting.vy`

File: README.md (hunk 1 above)
Comment (rbharath) [reply README.md]: Added this command in

**Pull request merged**